libr/io/p/io_debug.c

```diff
@@ -382,7 +381,7 @@ static int fork_and_ptraceme_for_mac(RIO *io, int bits, const char *cmd) {
 		}
 		// XXX: this is a workaround to fix spawning programs with spaces in path
 		if (strstr (argv[0], "\\ ")) {
-			argv[0] = r_str_replace (argv[0], "\\ ", " ", true);
+			argv[0] = r_str_arg_unescape (argv[0]);
```

Comment:
wrong uasge, this returns. int, just ignore the result because this function modifies the argv[0] already

Reply:
also, the function signature of this function is not in the .h so it segfaults hard